eclass/vdr-plugin-2.eclass

```diff
@@ -422,7 +429,7 @@ vdr-plugin-2_pkg_setup() {
 	fi
 
 	VDRVERSION=$(awk -F'"' '/define VDRVERSION/ {print $2}' "${VDR_INCLUDE_DIR}"/config.h)
-	APIVERSION=$(pkg-config --variable=apiversion vdr)
+	APIVERSION=$($PKG__CONFIG --variable=apiversion vdr)
```

Comment:
```suggestion
	APIVERSION=$(${PKG__CONFIG} --variable=apiversion vdr)
```

Reply:
@ulm all changes done and tested a dozen ebuilds with this eclass. Seems to work.